boat-maven-plugin/src/main/java/com/backbase/oss/boat/RadioMojo.java

```diff
@@ -0,0 +1,171 @@
+package com.backbase.oss.boat;
+
+import com.backbase.oss.boat.bay.client.ApiClient;
+import com.backbase.oss.boat.bay.client.api.BoatMavenPluginApi;
+import com.backbase.oss.boat.bay.client.model.BoatLintReport;
+import com.backbase.oss.boat.bay.client.model.UploadRequestBody;
+import com.backbase.oss.boat.bay.client.model.UploadSpec;
+import com.backbase.oss.boat.loader.OpenAPILoader;
+import com.backbase.oss.boat.loader.OpenAPILoaderException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationConfig;
+import feign.auth.BasicAuthRequestInterceptor;
+import io.swagger.v3.oas.models.OpenAPI;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+@Mojo(name = "radio", requiresDependencyResolution = ResolutionScope.RUNTIME, threadSafe = true)
+@Slf4j
+@Getter
+@Setter
+public class RadioMojo extends AbstractMojo {
+
+    @Parameter(property = "groupId", defaultValue = "${project.groupId}", required = true)
```

Comment:
These should also be documented. If you use `mvn help:describe -Dplugin=boat -Ddetail` (use `-Dplugin=groupId:artifactId:version` to view SNAPSHIT versions.) Then you can get output like:

```
boat:lint
  Description: API lint which provides checks for compliance with many of
    Backbase's API standards.
  Implementation: com.backbase.oss.boat.LintMojo
  Language: java

  Available parameters:

    failOnWarning (Default: false)
      Set this to true to fail in case a warning is found.

    ignoreRules
      List of rules ids which will be ignored.
```
Instead of `(no description available)`

Reply:
Done. Updated readme with what one will get `mvn help:describe -Dplugin=boat -Ddetail -Dgoal=radio` and a sample configuration to use it.